code/__HELPERS/path.dm

```diff
@@ -6,6 +6,7 @@
 
 /**
  * This is the proc you use whenever you want to have pathfinding more complex than "try stepping towards the thing"
+ * If no path was found, returns an empty list, which is important for bots like medibots who expect an empty list rather than nothing
```

Comment:
```suggestion
 * This is the proc you use whenever you want to have pathfinding more complex than "try stepping towards the thing".
 * If no path was found, returns an empty list, which is important for bots like medibots who expect an empty list rather than nothing.
```

The autodoc tool combines all these into one line, IIRC, so they need periods to not be a grmaatically incorrect sentence.

Reply:
Still need this one @Ryll-Ryll